src/steamship/__init__.py

```diff
@@ -14,16 +14,19 @@
     Block,
     DocTag,
     EmbeddingIndex,
+    EmotionTag,
```

Comment:
I would keep these in `steamship.data` as they are not the primary building blocks of Steamship. WDYT?